locales/bn/bn.json

```diff
@@ -143,127 +143,127 @@
+    "Detach Selected": "নির্বাচিত অংশ আলাদা করুন",
     "Details": "বিস্তারিত",
     "Disable": "নিষ্ক্রিয়",
-    "Djibouti": "বুলগেরিয়া",
-    "Do you really want to leave? You have unsaved changes.": "আপনি কি আসলেই চলে যেতে চান? তোমার পরিবর্তন অসংরক্ষিত.",
-    "Dominica": "রবিবার",
-    "Dominican Republic": "বুরুন্ডি",
-    "Done.": "হয়ে যাবে",
-    "Download": "ডাউনলোড করো",
-    "Download Receipt": "Download Receipt",
-    "E-Mail Address": "E-Mail Address",
-    "Ecuador": "ক্যাম্বোডিয়া",
+    "Djibouti": "জিবুতি",
+    "Do you really want to leave? You have unsaved changes.": "আপনি কি আসলেই চলে যেতে চান? আপনার পরিবর্তন অসংরক্ষিত।",
+    "Dominica": "ডমিনিকা",
+    "Dominican Republic": "ডোমিনিকান প্রজাতন্ত্র",
+    "Done.": "হয়ে গেছে",
+    "Download": "ডাউনলোড করুন",
+    "Download Receipt": "রিসিপ্ট ডাউনলোড করুন",
+    "E-Mail Address": "ই-মেইল ঠিকানা",
+    "Ecuador": "ইকুয়েডর",
     "Edit": "সম্পাদনা",
     "Edit :resource": "সম্পাদনা :resource",
     "Edit Attached": "সংযুক্ত সম্পাদনা করুন",
     "Editor": "সম্পাদক",
-    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীদের পড়তে তৈরি, এবং আপডেট করার ক্ষমতা আছে.",
-    "Egypt": "ক্যামেরুন",
-    "El Salvador": "কানাডা",
+    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীদের পড়তে তৈরি, এবং আপডেট করার ক্ষমতা রাখে।",
+    "Egypt": "মিশর",
+    "El Salvador": "এল সালভাদর",
     "Email": "ই-মেইল",
     "Email Address": "ই-মেইল ঠিকানা",
-    "Email Addresses": "Email Addresses",
+    "Email Addresses": "ই-মেইল ঠিকানা সমূহ",
     "Email Password Reset Link": "ইমেল পাসওয়ার্ড রিসেট লিংক",
-    "Enable": "সক্রিয় করো",
-    "Ensure your account is using a long, random password to stay secure.": "আপনার অ্যাকাউন্ট নিরাপদ থাকার একটি দীর্ঘ, র্যান্ডম পাসওয়ার্ড ব্যবহার করা হয় তা নিশ্চিত করুন.",
-    "Equatorial Guinea": "কেপ ভার্ডি",
-    "Eritrea": "কেমেন আইল্যাণ্ড",
-    "Estonia": "অধাতু",
+    "Enable": "সক্রিয় করুন",
+    "Ensure your account is using a long, random password to stay secure.": "আপনার অ্যাকাউন্ট নিরাপদ থাকার একটি দীর্ঘ, এলোমেলো পাসওয়ার্ড ব্যবহার করা হয় তা নিশ্চিত করুন।",
+    "Equatorial Guinea": "নিরক্ষীয় গিনি",
+    "Eritrea": "ইরিত্রিয়া",
+    "Estonia": "এস্তোনিয়া",
     "Ethiopia": "ইথিওপিয়া",
-    "ex VAT": "ex VAT",
-    "Extra Billing Information": "Extra Billing Information",
+    "ex VAT": "বিগত ভ্যাট",
+    "Extra Billing Information": "অতিরিক্ত বিল সংক্রান্ত তথ্য",
     "Extra confirmation is needed to process your payment. Please confirm your payment by filling out your payment details below.": "অতিরিক্ত নিশ্চিতকরণ আপনার পেমেন্ট প্রক্রিয়া প্রয়োজন হয়. নিচে আপনার পেমেন্ট বিবরণ পূরণ করে আপনার পেমেন্ট নিশ্চিত করুন.",
     "Extra confirmation is needed to process your payment. Please continue to the payment page by clicking on the button below.": "অতিরিক্ত নিশ্চিতকরণ আপনার পেমেন্ট প্রক্রিয়া প্রয়োজন হয়. নিচের বাটনে ক্লিক করে পেমেন্ট পাতা দয়া করে চালিয়ে যান.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.